src/features/game/events/landExpansion/plant.ts

```diff
@@ -128,15 +132,65 @@ export const getCropTime = (
     seconds = seconds * 0.75;
   }
 
+  // If within Basic Scarecrow AOE: 20% reduction
+  const isBasicCrop =
+    crop === "Sunflower" || crop === "Potato" || crop === "Pumpkin";
+
+  if (
+    isBasicCrop &&
+    isCollectibleBuilt("Basic Scarecrow", collectibles) &&
+    isWithinAOE("Basic Scarecrow", collectibles, plot)
+  ) {
+    seconds = seconds * 0.8;
+  }
+
   return seconds;
 };
 
+export function isWithinAOE(
```

Comment:
Consider moving this into our `collisionDetection` library - it will become useful for all types of resources 👍

Reply:
e.g.

export function isWithinAOE(areaOfImpact: Position, item: Position) {}

/*

Position is an existing type we have:
export type Position = {
  x: number;
  y: number;
  height: number;
  width: number;
};
*/